docker/Dockerfile

```diff
@@ -1,4 +1,34 @@
-FROM ubuntu:xenial
+FROM ubuntu:xenial as builder
+RUN apt-get update -y && apt-get install -y build-essential curl
```

Comment:
`--no-install-recommends` should also be in here

Reply:
I'll give this a try.

Reply:
Nope that doesn't work, and seeing as this is just the build image I'm ok with having some extra junk in there.